fvm/src/machine/default.rs

```diff
@@ -95,15 +95,14 @@ where
 
         // Load the built-in actors manifest.
         // TODO: Check that the actor bundle is sane for the network version.
-        let builtin_actors: Manifest = blockstore
-            .get_cbor(&builtin_actors)
-            .context("failed to load built-in actor index")?
-            .ok_or_else(|| {
-                anyhow!(
-                    "blockstore doesn't contain builtin actors index with CID {}",
-                    &builtin_actors
-                )
-            })?;
+        let maybe_builtin_actors = load_manifest(&blockstore, &builtin_actors);
```

Comment:
`builtin_actors` needs to be turned into an `Option<Cid>`. When `None`, we'll load it from the System Actor's state (after the `state_tree` has been constructed). When `Some`, we'll load it directly from the blockstore as we're doing now.

Reply:
We will need to copy the source for the system actor's state object into ref-fvm, just like we have for the init actor (and other implicitly coupled actors).

Reply:
done.